Resources/public/js/views/ez-fieldeditview.js

```diff
@@ -372,6 +380,14 @@ YUI.add('ez-fieldeditview', function (Y) {
             },
 
             /**
+             * Errors given by the serverSide validation.
+             *
+             * @attribute serverSideErrors
+             * @type Array of Y.eZ.FieldErrorDetails
+             */
+            serverSideErrors: {},
```

Comment:
API wise that's a bit weird. This attribute actually never receives an array or store an array. I would replace this attribute by a public method (`appendServerSideError`) and then the modified `serverSideErrorsChange` logic should be applied in this method